data/processSynapseJSON.ts

```diff
@@ -597,6 +620,9 @@ function processSynapseJSON(
         demographicsByParticipantID: demographicsByParticipantID as {
             [ParticipantID: string]: SerializableEntity;
         },
+        therapyByParticipantID: therapyByParticipantID as {
+            [ParticipantID: string]: SerializableEntity[];
+        },
```

Comment:
We may not want to attach this to the final json object. If we add the therapy data in `getSampleAndPatientData`, that may be enough for the frontend components to generate list of therapies for filtering purposes.

Reply:
Let's keep `therapyByParticipantID` attached to the final result for now to keep things consistent with the current implementation. We can remove it later if necessary.

Reply:
You are right, on adding the therapy data in `getSampleAndPatientData`,  the frontend components do generate list of therapies for filtering purposes. I'll remove the same from final json object.